src/main/scala/scalafix/sbt/ScalafixEnable.scala

```diff
@@ -57,10 +69,16 @@ object ScalafixEnable {
           ScalafixPlugin.autoImport.scalafixSemanticdb
         )
       )
-    } yield setting
+      settings <-
+        relaxScalacForScalafix ++
+          (if (!isSemanticdbEnabled) addSemanticdb else List())
+    } yield settings
 
-    val semanticdbInstalled = Compat.append(extracted, settings, s)
+    val scalafixReady = Compat.append(extracted, settings, s)
 
-    semanticdbInstalled
+    scalafixReady
   }
+
+  private def scalafixInvoked: Def.Initialize[Task[Boolean]] =
+    Def.task(executionRoots.value.exists(_.key == scalafix.key))
```

Comment:
Should this also check for `scalafixAll`?

Reply:
good catch! just added 9c91d70